sdk/keyvault/azure-security-keyvault-keys/src/key_encryption_algorithm.cpp

```diff
@@ -5,7 +5,17 @@
 #include "private/key_constants.hpp"
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
-
+// Disable deprecation warning
```

Comment:
I like this comment :)

Reply:
me too, i goth the code block from larry